cli-commands/shape/utils.js

```diff
@@ -0,0 +1,27 @@
+const request = require('request');
+
+async function verifyUrl(url) {
```

Comment:
I do not think we should have this check as it is too big - it makes full request to the url. Let git fail if the url is not OK